### PR TITLE
unlist list() -> vec, fixes #180

### LINKED ISSUE #180: Error in simulations when predInt is a list(). Error: 'list' object cannot be coerced to type 'double'
According to the manual predInt can be a list() if we want to simulate multiple intervals.
This code:
`predInt=list(c(168, 192, 1), c(336,360,1), c(503.8,527.8,1), c(627,696,1))`

is generating this error:

image

Reported by Jana Stojanova

### PR DESCRIPTION
**Pull request by Siel**
This commit aims to fix issue #180.

Here I'm modifying L1038, using unlist() to transform the list to a vector

### CODE REVIEW

**Comment by Siel** [comment]
Need some feedback before merging because I'm not sure if I might be breaking something else by doing this @mnneely

File: R/SIMrun.R [hunk 1]
```diff
@@ -533,8 +536,8 @@ SIMrun <- function(poppar, limits = NULL, model, data, split,
       covMean[which(names(covMean) %in% names(covariate$mean))] <- covariate$mean
       covMean <- unlist(covMean)
```
Comment (Siel) [inline R/SIMrun.R:537]: This is the specific change, other modifications are just formatting

File: R/SIMrun.R [hunk 2]
```diff
@@ -1032,7 +1035,7 @@ SIMrun <- function(poppar, limits = NULL, model, data, split,
       }
       predTimes <- sapply(predInt, function(x) rep(seq(x[1], x[2], x[3]), each = numeqt))
       # catenate columns into single vector
-      predTimes <- c(predTimes)
+      predTimes <- unlist(predTimes)
```
Comment (Siel) [inline R/SIMrun.R:1038]: This is the specific line that changed, other modifications are just formatting

Comment (mnneely) [reply R/SIMrun.R:1038]: We'd have to test it, but the code change seems logical and sensical.

**Pull request closed without merging**